R/pkg/R/DataFrame.R

```diff
@@ -1187,6 +1187,69 @@ setMethod("dapply",
             dataFrame(sdf)
           })
 
+#' gapply
+#'
+#' Apply a function to each group of a DataFrame. The group is defined by an input
+#' grouping column.
+#'
+#' @param x A SparkDataFrame
+#' @param func A function to be applied to each group partition specified by grouping
+#'        column of the SparkDataFrame.
+#'        The output of func is a local R data.frame.
+#' @param schema The schema of the resulting SparkDataFrame after the function is applied.
+#'               It must match the output of func.
```

Comment:
I do not know what is R styles. At least, the styles of `func` and `schema` are different.